tools/ci_build/github/pai/pai_get_thread.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash 
+
+agentName=$1
+finalCharacter=${agentName: -1}
+echo "agent name $agentName"
+echo "agent name final character : $finalCharacter"
+targetRender=$((finalCharacter+128))
+
+echo -e "\n ---- rocm-smi"
+rocm-smi
+
+echo -e "\n ---- rocm-smi --showpids"
+rocm-smi --showpids
+
+echo -e "\n ---- rocm-smi --showpidgpus"
```

Comment:
is this really useful?

Reply:
Yes, this is to record the CI environment, we can see if there are other threads occupying our target GPU, and we have used this to find the occupancy problems caused by PTE team.